components/admin/widgets/metadata/form/constants.js

```diff
@@ -0,0 +1,60 @@
+export const LANGUAGE_OPTIONS = [
+  { label: 'English', value: 'en' },
+  { label: 'Español', value: 'es' },
+  { label: 'Français', value: 'fr' },
+  { label: 'Português', value: 'pt' }
+];
+
+export const STATE_DEFAULT = {
+  step: 1,
+  stepLength: 1,
+  submitting: false,
+  loading: false,
+  form: {
+    authorization: '',
+    language: 'en',
+    info: {
+      widgetLinks: []
+    }
+  }
+};
+
+export const FORM_ELEMENTS = {
+  elements: {
+  },
+  validate() {
+    const elements = this.elements;
+    Object.keys(elements).forEach((k) => {
+      elements[k].validate();
+    });
+  },
+  isValid() {
+    const elements = this.elements;
+    const valid = Object.keys(elements)
+      .map(k => elements[k].isValid())
+      .filter(v => v !== null)
+      .every(element => element);
+
+    return valid;
+  }
+};
+
+export const SOURCE_ELEMENTS = {
```

Comment:
Are you using this?

Reply:
Not at the moment but it will be used as the number of fields here grows...